R/dot-net-wrapper.R

```diff
@@ -42,6 +42,14 @@ DotNetWrapper <- R6::R6Class(
       }
     },
 
+    readOnlyProperty = function(propertyName, value, returnValue) {

```

Comment:
@PavelBal This is a new method to wrap an actual readonly prop wthout having to duplicate all the code everywhere. We can use it here and there to simplify some of the existing code. (haven't done it)